docs/source/user_guide/index.rst

```diff
@@ -53,10 +53,10 @@ Lets generate a Gaussian pulse at focus, propagate it backwards by one Rayeligh
    laser = Laser(dimensions,lo,hi,num_points,laser_profile)
 
 ..  code-block:: python
-   :caption: Propagate the laser pulse backwards by one Rayeligh length.
+   :caption: By default, the laser antenna will emit on the focal plan. (Optional) Make the laser antenna emit outside the focal plan by one Rayleigh length.
```

Comment:
LASY does not really have an antenna (defined as a plane onto which we generate currents so as to create the desired electromagnetic wave). IMO, we should therefore avoid this word here, so I would have a similar comment to the point above. But I agree with you, it would be nice to make this a bit more clear: I mentioned this above.

Reply:
"By default, the values of the laser envelope are injected on the focal plan" does that sound better?